src/MinterGateway.sol

```diff
@@ -867,6 +872,16 @@ contract MinterGateway is IMinterGateway, ContinuousIndexing, ERC712Extended {
      */
     function _updateCollateral(address minter_, uint240 amount_, uint40 newTimestamp_) internal {
         uint40 lastUpdateTimestamp_ = _minterStates[minter_].updateTimestamp;
+        uint32 maxUpdateCollateralFrequency_ = maxUpdateCollateralFrequency();
+
+        if (maxUpdateCollateralFrequency_ != 0) {
+            uint32 timeSinceLastUpdate_ = uint32(newTimestamp_ - lastUpdateTimestamp_);
```

Comment:
newTimestamp can be < lastUpdateTimestamp_, check for it is below this block of code

Reply:
```solidity
        // MinterGateway already has more recent collateral update
        if (newTimestamp_ <= lastUpdateTimestamp_) revert StaleCollateralUpdate(newTimestamp_, lastUpdateTimestamp_);
````
is still there below this block of code, and has not changed.

Reply:
Good catch.